rasa_nlu/tokenizers/jieba_tokenizer.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Text
+
+from rasa_nlu.config import RasaNLUConfig
+from rasa_nlu.tokenizers import Tokenizer, Token
+from rasa_nlu.components import Component
+from rasa_nlu.training_data import Message
+from rasa_nlu.training_data import TrainingData
+
+
+class JiebaTokenizer(Tokenizer, Component):
+    name = "tokenizer_jieba"
+
+    provides = ["tokens"]
+
+    def train(self, training_data, config, **kwargs):
+        # type: (TrainingData, RasaNLUConfig, **Any) -> None
+
```

Comment:
I think it would be better if the component fails during training if the language is not set to a supported language. I don't quite like the fallback to whitespace tokenization.